### PR TITLE
fix(api_helpers): re-add falcon.api_helpers

### LINKED ISSUE #1902: Unannounced breaking change in renaming falcon.api_helpers
The `falcon.api_helpers` module used to contain several semi-public (not documented, but not underscored either) utilities, that people may be relying on.

During the 3.0 release cycle, `falcon.api_helpers` was renamed to `falcon.app_helpers` in order to reflect the `API` :arrow_right: `App` renaming.
However, the old name disappeared without any deprecation notice, and without any information in the 3.0 changelog notes. This is a bug.

In the spirit of SemVer, we cannot afford marking the change as deprecation in 3.0.1 either.
Let us make the name available as-is in 3.0.1, and mark as deprecated in 3.1; remove in 4.0.

### PR DESCRIPTION
**Pull request by CaselIT**
# Summary of Changes

Re-add removed api_helpers package

# Related Issues

closes #1902

### CODE REVIEW

File: docs/_newsfragments/1902.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Re-add ``api_helpers`` module that was renamed ``app_helpers``.
+This modules is to consider deprecated and will be removed in a future falcon version.
```
Comment (vytas7) [inline docs/_newsfragments/1902.rst:2]: > _This modules (sic) is to consider (sic) deprecated and will be removed in a future falcon version._

This sentence could be cleaned up a bit... maybe something along the lines of :arrow_down: 
> This module is considered deprecated, and will be removed in a future Falcon version.

File: falcon/api_helpers.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from .app_helpers import * # NOQA
+
+# TODO deprecate
+# import warnings
+# from .util.deprecation import DeprecatedWarning
+
+# warnings.warn('The api_helpers was renamed to app_helpers.', DeprecatedWarning)
```
Comment (vytas7) [inline falcon/api_helpers.py:7]: The `api_helpers` module or just `api_helpers`, IMHO at least.

**Review by vytas7 — CHANGES_REQUESTED** [review]
Looks great. :+1: 

I just nitpicked on the news fragment, it seems the second sentence therein could be improved a bit.

**Comment by CaselIT** [comment]
thanks, updated

**Review by vytas7 — APPROVED** [review]
:+1:

File: falcon/api_helpers.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from .app_helpers import * # NOQA
```
Comment (kgriffs) [inline falcon/api_helpers.py:1]: Nit: Should be two spaces before the `#`

**Review by kgriffs — APPROVED** [review]

File: falcon/api_helpers.py (hunk 3 above)
Comment (CaselIT) [reply falcon/api_helpers.py:1]: nit; using a formatter would take care of it :P

**Comment by floqqi** [comment]
Is there a chance of getting these changes released? It prevents us from upgrading to falcon 3 as we rely on the sentry sdk.

**Comment by vytas7** [comment]
Hi @floqqi! 
Yes, we are planning to cut 3.0.1 shorty, hopefully today or tomorrow.

We will be addressing https://github.com/falconry/falcon/pull/1911 in the same bugfix release.

**Pull request merged**